sites/public/pages/eligibility/age.tsx

```diff
@@ -94,6 +104,13 @@ const EligibilityAge = () => {
           <div className="form-card__pager">
             <div className="form-card__pager-row primary">
               <Button styleType={AppearanceStyleType.primary}>{t("t.next")}</Button>
+              <Button
+                onClick={handleSubmit(onClick)}
```

Comment:
To allow the user to click "view listings" without entering an age, I think all you'd have to do is get rid of `handleSubmit` (so that it's just `onClick={onClick}`)

Reply:
Looks like the onSubmit form overrides the onClick button status if I take away the handleSubmit.

Reply:
Ah gotcha, you're right - it looks like it does the form-submission validation for this button.

I looked this up (https://stackoverflow.com/questions/2825856/html-button-to-not-submit-form) and it looks like there's a simple little fix: set `type="button"` on the escape-hatch `Button`.

Reply:
Responding to your chat message here (re if a user enters an age and then clicks "view listings", shouldn't we use the age they entered to filter listings?). And yeah good point - we should use the age they entered (if they entered one). It seems like `handleSubmit(onClick)` does that for us, but it also comes with the submission validation (so the user can't leave age blank and click "view listings").

One way to get around that: if you use `document.getElementById("age").value` within `onClick`, you can get the value (if there is one) from the input field, and add it to the `eligibilityRequirements`. Something like:

```
  const onClick = () => {
    const age: number = document.getElementById("age").value
    if (age && isAgeValid(age)) {
      eligibilityRequirements.setAge(age)
    }
    void router.push(getFilterUrlLink(eligibilityRequirements))
  }
```

Note: when I use that code, I get the behavior I expect, but VSCode complains about `document.getElementById("age").value`: `Property 'value' does not exist on type 'HTMLElement'.`

Reply:
So informal survey but since I had a number of people (3) who weren't familiar with our site, I was able to survey them about having each step still validate and ask you to select or enter a value (depending on step of the questionnaire).  They all expected to input something on that step before selecting the View Listings button.  They said its a questionnaire, if I didn't want to answer that question I'd just go back or hit next.  So I'm thinking having it give the age warning is okay before View Listings.  Open to what others think.  @wrl and @anders-schneider?

Reply:
Also, if your objection is mostly about this age page in particular, the validation will be going away in #526 anyway.